### PR TITLE
reference-manual:: boards:: add apalis-imx6 flashing procedure

### PR DESCRIPTION
**Pull request by vanmaegima**
Signed-off-by: Vanessa Maegima <vanessa.maegima@foundries.io>

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 1fee40ae773376816c76781a8f67021d58bb73bd are browesable at: https://ci.foundries.io/projects/fio-docs/builds/861/docs/artifacts/html/index.html

File: source/reference-manual/boards/apalis-imx6.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
+Apalis iMX6
```
Comment (ricardosalveti) [inline source/reference-manual/boards/apalis-imx6.rst:1]: Just extend this piece to say that this is for Apalis iMX6 with the Ixora Carrier Board (e.g. https://www.toradex.com/products/carrier-board/ixora-carrier-board), as apalis itself is just the module.

**Review by Tim-Anderson — APPROVED** [review]
Add the comments suggested by Ricardo and it LGTM

**Review by angolini — APPROVED** [review]

**Comment by doanac** [comment]
Docs for 28960c893731597f61d110370fbf24424fdb2d8a are browesable at: https://ci.foundries.io/projects/fio-docs/builds/865/docs/artifacts/html/index.html

**Review by ricardosalveti — APPROVED** [review]
LGTM, thanks for doing this one.

**Pull request merged**